src/app/modules/core/advertisement/advertisement.module.ts

```diff
@@ -51,6 +51,6 @@ const advertisementsComponent = [
 @NgModule({
   declarations: [...advertisementsComponent],
   imports: [CommonModule, AdvertisementRoutingModule, SharedModule],
-  providers: [AdvertisementService, { provide: LOCALE_ID, useValue: 'en-EN' }],
+  providers: [AdvertisementService, { provide: LOCALE_ID, useValue: 'it-IT' }],
```

Comment:
@CarmeloB4 non so come faccia a funzionare 😂 ma da errore in console `Missing locale data for the locale "it-IT"` e la view degli advertisements infatti si carica male

Dovresti importare e registrare il locale IT nell'app module come abbiamo fatto per l'esercizio del meteo e togliere da qui il provider per metterlo globalmente nel progetto

```
import localeIt from '@angular/common/locales/it';
import { registerLocaleData } from '@angular/common';

registerLocaleData(localeIt);
```